policy/custom/definitions/policy/LA-Microsoft.AzureRecoveryVault-Backup/azurepolicy.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "Deploy Diagnostic Settings for Recovery Vault for Azure Backup Events",
+    "mode": "all"
```

Comment:
mode should be `indexed`.  See:  https://docs.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#mode